web/src/pages/Configuration/Moderation/index.tsx

```diff
@@ -112,29 +122,34 @@ function ModerationUsersList() {
       /> */}
       <div className='user-list__wrapper'>
         <table className='user-list__table'>
-          <thead className='user-list__table--header'>
-            <tr className='user-list__table--header--row'>
-              <th className='user-list__table--header--cell'>{t('user.email')}</th>
-              <th className='user-list__table--header--cell'>{t('user.name')}</th>
-              <th className='user-list__table--header--cell'>{t('moderation.role')}</th>
-              <th className='user-list__table--header--cell'>{t('moderation.verified')}</th>
-              <th className='user-list__table--header--cell'>{t('moderation.actions')}</th>
+          <thead className='user-list__table-header'>
+            <tr className='user-list__table-header-row'>
+              <th className='user-list__table-header-cell'>{t('user.email')}</th>
+              <th className='user-list__table-header-cell'>{t('user.name')}</th>
+              <th className='user-list__table-header-cell'>{t('moderation.role')}</th>
+              <th className='user-list__table-header-cell'>{t('moderation.verified')}</th>
+              <th className='user-list__table-header-cell'>{t('moderation.actions')}</th>
             </tr>
           </thead>
-          <tbody className='user-list__table--body--row'>
+          <tbody className='user-list__table-body'>
               {users.map((user, idx) => (
-                <tr className='user-list__table--body--cell' key={idx}>
-                  <td className='user-list__table--body--cell'><Link href={`/p/${user.username}`}>{user.email}</Link></td>
-                  <td className='user-list__table--body--cell'><Link href={`/p/${user.username}`}>{user.name}</Link></td>
-                  <td className='user-list__table--body--cell'>{t(`roles.${user.role}`)}</td>
-                  <td className='user-list__table--body--cell'>
+                <tr className='user-list__table-body-row' key={idx}>
+
+                  <td className='user-list__table-body-cell--email'>
+
+                      <Link href={`/p/${user.username}`}>{getEmailPrefix(user.email)}</Link> 
```

Comment:
why you want only the email prefix? isnt this only visible for admins? we have to trust admins of our networks.